index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link  href="style.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Merhia Rahmani - Portfolio</title>
+</head>
+
+<body>
+        <header class="container d-flex">
+
+                <a href="#home" class="logo">Merhia Rahmani</a>
+                <nav>
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>
+        </header>
+
+    <section class="container hero-wrapper">
+        <div class="hero-container">
+            <h1 class="headline d-flex">Hey There.<br>I'm Mehria</h1>
+            <h2 class="job-title d-flex">I am a web full stack developer </h2>
+            <p class="job-description d-flex">I can help you build a product, feature or website.Look through some of my work and experience! If you like what you see and have a project you need coded, do not hestiate to contact me.</p>
+            
+        </div>
+        <div class="links d-flex">
+            <ul>
+                <li>
+                    <a href="https://github.com/Mehria-Rahmani/Hello-Microverse.git" id="git" target="blank"><img src="assets/github.svg" alt="github"></a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/mehria-rahmani-379211222" id="linkedin" target="blank"><img src="assets/linkedin.svg" alt="linkedin"></a>
+                </li>
+
+                <li>
+                    <a href="https://twitter.com/MehriaRahmani?t=QwKwWXTVBZthv4sUrOXtfQ&s=08" id="twitter" target="blank"><img src="assets/twitter.svg" alt="twitter"></a>
+                </li>
+                <li>
+                    <a href="http://medium.com" id="medium" target="blank"><img src="assets/medium.svg" alt="linkedin"></a>
+                </li>
+                <li>
+                    <a href="http://angellist.com" id="angel-list" target="blank"><img src="assets/angellist.svg" alt="angel-list"></a>
+                </li>
+
+            </ul>
+           
+        </div>    
+
+    </section>
+  
+    
+  
+   
+
```

Comment:
- [ **OPTIONAL** ] I think it would be better to remove all empty lines to make your code look cleaner and more organized.